src/PointwiseFunctions/Elasticity/ConstitutiveRelations/CubicCrystal.hpp

```diff
@@ -47,20 +51,19 @@ namespace Elasticity::ConstitutiveRelations {
  *
  * \f[
  * E = \frac{(c_{11} + 2c_{12})(c_{11} - c_{12})}{c_{11} + c_{12}}, \quad
- * \nu = (1 + \frac{c_{11}}{c_{12}})^{-1}, \quad
+ * \nu = \left(1 + \frac{c_{11}}{c_{12}}\right)^{-1}, \quad
  * \mu = c_{44}
  * \f]
  *
  * The stress-strain relation then reduces to
  *
  * \f[
- * T^{ij} = -\delta^{ij} \left( \left( c_{11} - c_{12} - 2*c_{44} \right) S^{ij}
- * + c_{12} \mathrm{Tr}(S) \right) - 2*c_{44} S^{ij}
- * \f]
+ * T^{ij} = -\delta^{ij} \left[ (c_{11} - c_{12} - 2 c_{44})  S_{ii} + c_{12}
+ * \mathrm{Tr}(S) \right] - 2 c_{44} S^{ij} \f]
```

Comment:
we think it would be most clear to use `begin{cases}` and `end{cases}` for the `i = j` and `i \neq j` cases for this equation